Dockerfile

```diff
@@ -12,4 +12,6 @@ RUN npm install
 
 COPY . .
 
+WORKDIR /
```

Comment:
Why do we need to specify the `WORKDIR` at this point? The default one is already the `/` and the ENTRYPOINT is using absolute paths.

Reply:
Without this, the image boots at `/app`, but later the script [removes this app folder](https://github.com/0xPolygonHermez/zkevm-bridge-ui/blob/develop/scripts/deploy.sh#L76) resulting in this error when you access the container:

```bash
(unknown) #
sh: getcwd: No such file or directory
```

`getcwd` tries to get the current working directory and if the directory is not found it complains.